common/culture/cultures/wc_gnomic.txt

```diff
@@ -1,11 +1,16 @@
 ﻿gnome = {
 	color = rgb { 222 135 26 }
 	
-	ethos = ethos_egalitarian
+	ethos = ethos_bureaucratic
```

Comment:
> Bureaucratic
> No culture lasts longer than its oldest record, and this culture believes in accounting so meticulous as to guarantee immortality.

Dunno, that doesn't sound like it would fit gnomes

https://wowpedia.fandom.com/wiki/Gnome#Culture

> As noted above, much of gnomish philosophy is focused on forward-thinking, inventive concepts, and little thought has ever been given to history or non-scientific records.

I think this one would git them better, imo

> Communal
> This culture values the bonds of community above all else, fostering great loyalty and dedication by working together towards common goals.

> From that is known, gnomes do value and follow the hierarchy, but they also understand the hierarchy as the masters' duty over the servants, to care for them and help them. Some senior gnomish engineers may actually have less parts, metals and similar resources than an ordinary engineer, as they tend to share them with the followers. Also, the higher rank is most easy to earn by teaching others, while this, of course, requires to be a master on a subject. When goblins say they hate hierarchy as the master first cares about himself (so who would want to be the servant), they seem talking in two completely different languages. Gnome groups are small and form with difficulty over a long time. However when once established, they usually last for life, and a gnome reasonably expects much more support and cooperation from the team-mates than a goblin would.

Reply:
Great point, thanks for the correction!